examples/dms/find_queue_group.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+"""
+Find a Queue Group
+"""
+import openstack
+
+openstack.enable_logging(True)
+conn = openstack.connect(cloud='otc')
+
+queue = 'queue-name-or-id'
+queue = conn.dms.find_queue(name_or_id=queue)
+group = 'group-name-or-id'
+raw = conn.dms.find_group(queue, name_or_id=group)
```

Comment:
to minimize issues on user side, it is more usefull to declare the variables explicitely:
find_group(queue=queue, name_or_id=group)